ir/shared/src/main/scala/org/scalajs/ir/Position.scala

```diff
@@ -20,16 +20,18 @@ final case class Position(
     /** Zero-based column number. */
     column: Int
 ) {
-  def show: String = s"$line:$column"
-
-  def isEmpty: Boolean = {
+  private val _isEmpty: Boolean = {
     def isEmptySlowPath(): Boolean = {
       source.getScheme == null && source.getRawAuthority == null &&
         source.getRawQuery == null && source.getRawFragment == null
     }
     source.getRawPath == "" && isEmptySlowPath()
   }
 
+  def show: String = s"$line:$column"
+
+  def isEmpty: Boolean = _isEmpty
```

Comment:
Have you considered making `source` nullable?

Reply:
I wish I could do that, but `source` is a very much a public field, and I don't want to expose a nullable field in our public API.